# Publication_Sigma54
The data and scripts that accompany the publication Khan EA, Ruckert-Reed C, Dahiya GS, Tietze L, Fages-Lartaud M, Busche T, Kalinowski J, Shingler V, Lale R. High-resolution mapping of Sigma Factor DNA Binding Sequences using Artificial Promoters, RNA aptamers and Deep Sequencing.

DNA_Sequencing folder, Splitting the DNA-sequenced ARES library into multiple files.

Plasmid_maps folder • Cloned Mango aptamer in pSEVA2311 backbone • Cloned ARES (N200) in pHH100-dBroccoli backbone • Map of ARES with adapter • Cloned dBroccoli aptamer in pHH100 backbone

RNA_Sequencing folder, Unique library components mapped to RNA and TSS are written in capital letters.

ARES_New_IDs folder, The IDs used in the DNA and RNA sequence files differ from those used in the SRA submission file. To assist researchers, I have included both the SRA and raw data IDs in these files for easy reference.

Data_Analysis_Script folder, • Python script for the finding of promoter motifs • Perl script for marking the TSS

Supplementary folder contain figures (• Importance of ΔA2-DmpR for P. putida σ54-Dependent IVT reaction • Distribution of single motif within a 97 bp region upstream of the TSS in the highest expressing ARES subset • Distribution of single motif within the 40 bp region upstream of the TSS in the highest expressing ARES subset.), tables (• Primer sequences • Sequence of the shortlisted ARES (18) for IVT validation • Significant annotation of IVT-generated σ54-RNA polymerase binding motifs in the upstream regions of genes.) and Benchling links of the plasmids used in this study
